### ISSUE TITLE
Tab-completion of keyword arguments makes paranthesis

### ISSUE THREAD

**Issue by skjerns** [spyder-ide/spyder#12333]
## Problem Description

If I press <tab> to autocomplete a keyword, instead of an `=` I get a `()` as a suggestion

### What steps reproduce the problem?
```Python
def test(argument=True):
    pass

# now write
test(argu<tab>

# expected
test(argument=

# instead
test(argument()
```

This worked fine in previous version. Is this a bug on my side?

* Spyder version: 4.1.2
* Python version: 3.7
* Operating System name/version: Win 10

**Comment by arteagac** [comment]
I am facing the same issue. 
Spyder 4.2.0.dev0 (Commit: 755acb75f) 
Python 3.7.7 64-bit | Qt 5.9.6 | PyQt5 5.9.2 | Linux 5.4.0-26-generic

**Comment by goanpeca** [comment]
I can confirm this bug.

![Screen Shot 2020-05-04 at 22.29.37.png](https://images.zenhubusercontent.com/560c1b40638e718c995d70b3/7137bf96-96b4-45b8-b22c-4a4f993750cc)

**Comment by ccordoba12** [comment]
I think this is a bug in the PyLS.

@steff456, please take a look at this one.

**Comment by steff456** [comment]
I checked out the error in PyLS, and it seems that this is fixed by upgrading jedi to 0.17. So this issue should be fixed when PR https://github.com/spyder-ide/spyder/pull/12792 is merged.

![jedi](https://user-images.githubusercontent.com/20992645/82383131-d4d3b100-99f2-11ea-871c-650380cb0668.gif)
